pkg/piecestore/psserver/psdb/testdata/db.v0.sql

```diff
@@ -0,0 +1,9 @@
+PRAGMA foreign_keys=OFF;
+BEGIN TRANSACTION;
```

Comment:
Ah, also forgot that we want each table to contain some data. Otherwise it will be easy to miss migrations that need to transform data.